srv/salt/ceph/rgw/dashboard.sls

```diff
@@ -10,7 +10,11 @@ configure dashboard rgw secret key:
 
 configure dashboard rgw port:
   cmd.run:
+{% if {{ salt['pillar.get']('rgw_init', 'default') }} == "default-ssl" %}
```

Comment:
nit: `{{ pillar.get('rgw_init', 'default') }}` is less verbose, and should also work.